resource/menu/server/sv_main_page.lua

```diff
@@ -189,6 +189,68 @@ RegisterNetEvent('txAdmin:menu:showPlayerIDs', function(enabled)
   end
 end)
 
+---Stores data needed for blips
+---@type { [serverID]: {coords: {x:number,y:number,z:number,h:number}, name: string, blipsEnabled: boolean, foundLastCheck: boolean} }
+local playerBlipsData = {}
+local BLIPS_REFRESH_TIMEOUT = 2000
+local intervalYieldLimit = 50
+
+RegisterNetEvent('txAdmin:menu:showPlayerMapBlips', function(enabled)
+  local src = source
+  local allow = PlayerHasTxPermission(src, 'menu.viewids')
+  if allow then
+    if type(playerBlipsData[tostring(src)]) == 'table' then
+      playerBlipsData[tostring(src)].blipsEnabled = not playerBlipsData[tostring(src)].blipsEnabled
+    else
+      playerBlipsData[tostring(src)] = {
+        blipsEnabled = true
+      }
+    end
+    TriggerEvent("txaLogger:menuEvent", src, "showPlayerMapBlips", allow, playerBlipsData[tostring(src)].blipsEnabled)
+    TriggerClientEvent('txAdmin:menu:refreshPlayerBlips', src, {}, playerBlipsData[tostring(src)].blipsEnabled)
+  end
```

Comment:
```suggestion
  local src = source
  local strSource = tostring(src)
  local allow = PlayerHasTxPermission(src, 'menu.viewids')
  if allow then
    if type(playerBlipsData[strSource]) == 'table' then
      playerBlipsData[strSource].blipsEnabled = not playerBlipsData[strSource].blipsEnabled
    else
      playerBlipsData[strSource] = {
        blipsEnabled = true
      }
    end
    TriggerEvent("txaLogger:menuEvent", src, "showPlayerMapBlips", allow, playerBlipsData[strSource].blipsEnabled)
    TriggerClientEvent('txAdmin:menu:refreshPlayerBlips', src, {}, playerBlipsData[strSource].blipsEnabled)
  end
  ```
  I think the code is better if there are not many tostrings 😄